### PR TITLE
#28 Test#5

### LINKED ISSUE #28: Test #5
User can delete more than one comment.

### PR DESCRIPTION
**Pull request by oteplova**
Test#5: User can delete multiple comments (> 1)

Closes #28

### CODE REVIEW

File: src/main/java/net/comments/ft/Test2.java [hunk 1]
```diff
@@ -17,9 +17,9 @@
     @Test
     public void test() {
         page.open();
-        int commentNumber = 5;
+        int[] commentNumber = {5};
```
Comment (extsoft) [inline src/main/java/net/comments/ft/Test2.java]: Why do you use an array here?

File: src/main/java/net/comments/ft/Test3.java [hunk 2]
```diff
@@ -16,12 +16,12 @@
     @Test
     public void test() {
         page.open();
-        int commentNumber = 1;
+        int[] commentNumber = {1};
```
Comment (extsoft) [inline src/main/java/net/comments/ft/Test3.java]: Why do you use an array here?

File: src/main/java/net/comments/ft/Test4.java [hunk 3]
```diff
@@ -19,11 +19,11 @@ public Test4() {
     public void test() {
         page.open();
         CommentTable commentTable = page.getCurrentComments();
-        int commentNumber = 1;
+        int[] commentNumber = {1};
```
Comment (extsoft) [inline src/main/java/net/comments/ft/Test4.java]: Why do you use an array here?

File: src/main/java/net/comments/ft/Test5.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    Test5() {
+        this.page = new BCommentPage(driver);
+    }
+
+    @Test
+    public void test() {
+        page.open();
+        CommentTable commentTable = page.getCurrentComments();
+        int[] commentNumbers = {2, 5};
+        page.getCurrentComments().selectComment(commentNumbers);
```
Comment (extsoft) [inline src/main/java/net/comments/ft/Test5.java]: Why do you use an array here? If you need to select 2 comments, use method twice.

File: src/main/java/net/comments/objects/BCommentTable.java [hunk 5]
```diff
@@ -13,8 +13,11 @@ public BCommentTable(WebDriver driver) {
         this.driver = driver;
     }
 
-    public void selectComment(int commentNumber) {
-        this.driver.findElement(By.xpath(String.format("//*[@name = \"SelectedId\" and @value = \"%s\"]", commentNumber + 1))).click();
+    public void selectComment(int[] commentNumbers) {
```
Comment (extsoft) [inline src/main/java/net/comments/objects/BCommentTable.java]: If you want to have a method to select several comments, create it. Don't try  to have [GOD methods](https://en.wikipedia.org/wiki/God_object).

**Review by extsoft — CHANGES_REQUESTED** [review]

**Comment by oteplova** [comment]
@extsoft I've fixed the comments. Please check.

File: src/main/java/net/comments/objects/BCommentTable.java [hunk 6]
```diff
@@ -45,4 +45,11 @@ public String getCommentState(int commentNumber) {
         }
         return commentsList;
     }
+
+    public void selectComments(int[] commentNumbers) {
+        for (int commentNumber : commentNumbers) {
+            this.driver.findElement(By.xpath(String.format("//*[@name = \"SelectedId\" and @value = \"%s\"]", commentNumber + 1))).click();
```
Comment (extsoft) [inline src/main/java/net/comments/objects/BCommentTable.java]: @oteplova Use `this. selectComment(commentNumber)` instead of.

**Review by extsoft — CHANGES_REQUESTED** [review]

**Comment by oteplova** [comment]
@extsoft I've fixed the comment. Please review.

**Comment by oteplova** [comment]
@extsoft  Could you please check my chsnges?

**Review by extsoft — APPROVED** [review]
@oteplova great job

**Pull request merged**